global/source-zaza/tox.ini

```diff
@@ -89,4 +89,4 @@ commands = {posargs}
 
 [flake8]
 # E402 ignore necessary for path append before sys module import in actions
-ignore = E402,W504
+ignore = E402,W503
```

Comment:
are you removing `W504` on purpose withing this PR or is it by accident?

Reply:
Nope, it's on purpose.